components/mpas-ocean/cime_config/buildnml

```diff
@@ -429,7 +430,7 @@ def buildnml(case, caseroot, compname):
         if ocn_ic_mode == 'spunup':
             logger.warning("WARNING: The specified compset is requesting ocean ICs spunup from a G-case")
             logger.warning("         But no file available for this grid.")
-        if ocn_ismf == 'data':
+        if ocn_glc_ismf_coupling == 'data_mpaso':
```

Comment:
Great, glad you caught this one, too.  I needed this change in my test-merge branch.